detox/src/devices/common/drivers/android/exec/ADB.js

```diff
@@ -100,20 +100,15 @@ class ADB {
     return isInstalled;
   }
 
-  async install(deviceId, apkPath) {
-    apkPath = `"${escape.inQuotedString(apkPath)}"`;
-
+  async install(deviceId, _apkPath) {
+    const apkPath = escape.inQuotedString(_apkPath);
     const apiLvl = await this.apiLevel(deviceId);
+    const command = (apiLvl >= 23)
+      ? `install -r -g -t ${apkPath}`
+      : `install -rg ${apkPath}`;
+    const result = await this.adbCmdSpawned(deviceId, command, { timeout: 45000, retries: 3});
```

Comment:
Is `45000` a good magic number? Not that I want to make it configurable, but I would like to be sure it is reliable... what's the average duration of installation?

Reply:
A proper installation of a large app takes no more than a few seconds, actually. Factoring in slow networks and such, 45 is quite a decent timeout. You think it should be defined more explicitly?

Reply:
Let's add a TODO pls.